content/early-years-teaching-training.md

```diff
@@ -0,0 +1,104 @@
+      - "Graduate early years teacher training"     
+  EY-graduate-employment:
+    name: feature_table
+    arguments:
+      - Cost: "£7,000"
+        Salary: "Provided by employer"
+        Funding available: "Yes"
+        Length of training: "Typically 1 academic year"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate employment-based early years teacher training"
+  EY-self-funded:
+    name: feature_table
+    arguments:
+      - Cost: "Self-funded"
+        Salary: "No"
+        Funding available: "No"
+        Length of training: "3 months"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate Assessment-Only training"
+---
+
+## Become an early years teacher
+ 
+As an early years teacher, you'll work with young children up to the age of 5. Your input will have a lasting and positive effect on their development and wellbeing.
+ 
+By training to be an early years teacher, you’ll use your knowledge and skills to help young children enjoy high standards of teaching. 
+
+The early years are critical in children's development. You’ll be opening their minds to new ideas every day.
+ 
+###How to become an early years teacher
+ 
+You’ll need to achieve Early Years Teacher Status (EYTS). This is different from Qualified Teacher Status (QTS), which is the equivalent for teaching children aged 3 to 18. 
+
+Your training will enable you to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+ 
+To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
+
+* Maths grade 4 (C)
+* English grade 4 (C)
+* Science grade 4 (C)
+
+If you do postgraduate early years teacher training, you'll need a 2:2 bachelor’s degree with honours or higher.
+ 
+[Contact one of our early years training providers to apply for a place](https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations).
+
+## Ways to train
+
+$EY-undergraduate$
+
+A full-time bachelor’s degree in an early childhood-related subject leading to EYTS. [Undergraduate tuition fee loans are available from Student Finance England
+(SFE)](https://www.gov.uk/student-finance).
+
+$EY-graduate$
+ 
+Bursaries are available, including:
+
+ * £5,000 if you have a first class degree
+ * £4,000 if you have a 2:1 degree
+ * £2,000 if you have a 2:2 degree
+ 
+$EY-graduate-employment$
+
+This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards). Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
```

Comment:
```suggestion
This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards). 

Funding of up to £14,000 is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train.
```